### PR TITLE
Fallback CSS refresh

### PR DESCRIPTION
**Pull request by WumboSpasm**
Bringing formatting improvements from the Metal themes to the fallback CSS. Changes affect mainly margin/padding values, with any major visual tweaks going to the center boxes throughout the launcher. Will have compatibility issues of varying levels with older themes (they will need to be updated).

### CODE REVIEW

File: static/window/styles/fancy.css [hunk 1]
```diff
@@ -426,7 +424,7 @@ body {
 
 /* ------ About ------ */
 .about-page__section__title {
-  background-color: var(--layout__tertiary-background);
+  /* background-color: var(--layout__tertiary-background); */
```
Comment (colin969) [inline static/window/styles/fancy.css:427]: Have these been left in on purpose?

Comment (WumboSpasm) [reply static/window/styles/fancy.css:427]: In all honesty, I don't know why I kept them lol. If it's a problem feel free to get rid of them.

File: static/window/styles/core.css [hunk 2]
```diff
@@ -52,7 +52,9 @@ body {
   width: 1rem;
   height: 1rem;
 }
-
+.icon--magnifying-glass {
```
Comment (TBubba) [inline static/window/styles/core.css:55]: This applies to all magnifying icons and not just the one in the search bar.

Use something like ``.header__search__icon`` or ``.header__search__icon.icon--magnifying-glass`` instead (and remember to move it)

File: static/window/styles/core.css [hunk 3]
```diff
@@ -201,6 +207,7 @@ body {
   position: relative;
   display: flex;
   width: 100%;
+  border: none !important;
```
Comment (TBubba) [inline static/window/styles/core.css:210]: Don't use ``!important``

**Pull request merged**